appintro/src/main/res/values/dimen.xml

```diff
@@ -15,7 +15,8 @@
     <dimen name="appintro_button_minwidth">48dp</dimen>
 
     <dimen name="appintro_indicator_inset">3dp</dimen>
-    <dimen name="appintro_indicator_size">10dp</dimen>
+    <dimen name="appintro_unselected_indicator_size">10dp</dimen>
```

Comment:
Please keep those two together

Reply:
What does together mean?

Reply:
Just have a single:
```
    <dimen name="appintro_indicator_size">10dp</dimen>
```

And not two `appintro_unselected_indicator_size` and `appintro_selected_indicator_size`

Reply:
No. Disagreed.

Reply:
> No. Disagreed.

Do you have arguments?

Reply:
Having 2 separate dimensions is for changing the size of the selected indicator dot. We can make it bigger if we want this way

Reply:
Here are the arguments ^

Reply:
Something just like this:. 
https://drive.google.com/file/d/18RsCpEWXe8CCYqu3gNcVjfVdeAB1tkOA/view?usp=drivesdk